### PR TITLE
changed strings to use the format() method

### PR DESCRIPTION
**Pull request by andy5995**

### CODE REVIEW

File: trivia.py [hunk 1]
```diff
@@ -142,21 +142,21 @@ def signedOn(self):
         Actions to perform on signon to the server.
         '''
         self.join(self._game_channel)
-        self.msg('NickServ', 'identify %s' % config.IDENT_STRING)
-        print("Signed on as %s." % (self.nickname,))
+        self.msg("NickServ", "identify {}".format(config.IDENT_STRING))
+        print("Signed on as {}.".format(self.nickname))
```
Comment (andy5995) [inline trivia.py]: deleted comma?

File: trivia.py [hunk 2]
```diff
@@ -142,21 +142,21 @@ def signedOn(self):
...
             self._gmsg("For how to use this bot, just say ?help or")
-            self._gmsg("%s help." % self.nickname)
+            self._gmsg("{} help.".format(self.nickname))
 
     def joined(self, channel):
         '''
         Callback runs when the bot joins a channel
         '''
-        print("Joined %s." % (channel,))
+        print("Joined {}.".format((channel)))
```
Comment (andy5995) [inline trivia.py]: 2nd deleted comma

File: trivia.py [hunk 3]
```diff
@@ -489,11 +490,11 @@ def __init__(self, nickname=config.DEFAULT_NICK):
         self.lineRate = config.LINE_RATE
 
     def clientConnectionLost(self, connector, reason):
-        print("Lost connection (%s)" % (reason,))
+        print("Lost connection ({})".format(reason))
```
Comment (andy5995) [inline trivia.py]: 3rd deleted comma

File: trivia.py [hunk 4]
```diff
@@ -489,11 +490,11 @@ def __init__(self, nickname=config.DEFAULT_NICK):
...
         connector.connect()
 
     def clientConnectionFailed(self, connector, reason):
-        print("Could not connect: %s" % (reason,))
+        print("Could not connect: {}".format(reason))
```
Comment (andy5995) [inline trivia.py]: last deleted comma

File: trivia.py (hunk 1 above)
Comment (rawsonj) [inline trivia.py]: That comma is there I think just to make it a tuple, which looking at it isn't required I think.

It can be deleted.

**Pull request merged**